packages/webapp/src/webview/ui/components/Header/NavigationButtons/NavigationButtons.tsx

```diff
@@ -155,18 +155,23 @@ export function ShareButton() {
                                                 />
                                             </button>
                                         </CopyToClipboard>
+
+                                        <div className="sharable-link__divider-vertical"></div>
+
+                                        <CopyToClipboard text={copyInstructions} onCopy={toggleCopied}>
```

Comment:
Looks like [`react-copy-to-clipboard`](https://www.npmjs.com/package/react-copy-to-clipboard) has also the option to copy as HTML. Can we do this as we have clickable links in the text?
See: https://www.npmjs.com/package/copy-to-clipboard#api

Reply:
This gets problematic if people paste the text into something that cannot properly parse HTML. 

I'd rather we simply put plaintext link onto the clipboard, and then when users paste the text, if they are pasting into something fancy like Outlook, then Outlook itself will do the magic of wrapping the link into actual link. 

If users are pasting this in some chat or some other low-tech solution, then they will get a bunch of HTML code.

Reply:
For this PR, lets go ahead with text. AT least we will have a working version.
I will play with html option little bit later. SNOW is very important use case for this feature